practical/swarm_demo/install.sh

```diff
@@ -5,19 +5,28 @@
 #
 # USAGE: chmod +x ./install.sh && ./install.sh
 #
-# DESCRIPTION: Tears down an existing Swarm hosts and
-#              provisions a fresh cluster.
-
+# DESCRIPTION: Creates a Docker Swarm cluster with a discovery service node (Consul),
+#              a master and two slave nodes and runs the CA674 demo app on top of it.
+#              The CA674 demo is comprised of a back-end store (redis) and front-end
+#              www implementation (python) scaled to initially two instances.
+#              A load balancer is used to randomize access to each front-end container
+#              and unify the entry point of the app.
+#
 #===================================================================================
 
-VM=local
+function teardown_swarm_machines {
+  echo "Removing old Swarm machines..."
+  docker-machine stop keystore
```

Comment:
It's not how it works in my machine. When I run the stop command it does not matter if the command fails or not, it will keep executing to the end. Also, if you don't stop the machine beforehand it might break the virtualbox environment and then you'll have to manually remove the files and do a docker-machine rm <machine> which is pain.